scripts/format-values/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	inputFilePath  string
+	chartVersion   string
+	outputFilePath string
+)
+
+const (
+	NEW_VERSION = "v1.8.0"
+	OPERATOR    = "operator"
+	STARROCKS   = "starrocks"
+)
+
+var _starrocksKeys = []string{"nameOverride", "initPassword", "timeZone", "datadog", "starrocksCluster",
+	"starrocksFESpec", "starrocksCnSpec", "starrocksBeSpec", "secrets", "configMaps", "feProxy"}
+
+var _operatorKeys = []string{"global", "timeZone", "nameOverride", "starrocksOperator"}
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, `
+This tool is used to upgrade or downgrade the version of values.yaml for kube-starrocks chart.
+
+If the chart version is less than v1.8.0, the values.yaml is in the following format:
+aaa: bbb
+xxx: yyy
+
+If the chart version is greater than or equal to v1.8.0, the values.yaml will be changed to the following format:
+operator:
+  aaa: bbb
+starrocks:
+  xxx: yyy
+
+If you want to upgrade the version of values.yaml, you can run the following command:
```

Comment:
is it possible to auto detect the format if version is not provided?

Reply:
yes, this tool can detect the version. `--version` is target version to upgrade or downgrade

Reply:
maybe rename to `target_version`?